### PR TITLE
Adjusted replica count

### PR DESCRIPTION
**Pull request by xpr3sso**
Modified the approach for getting replica and in-sync replica count

### CODE REVIEW

**Comment by tchiotludo** [comment]
Can you help me a little bit ?
Is this just style or fixing a wrong value ?

**Comment by xpr3sso** [comment]
Yes, ofc. I encountered this when testing a topic (10 partition, replication 1) on a 4 broker cluster, returning replication factor and in-sync relpica of 3. The code was checking for unique nodes the topic's partitions were distributed on, in my case broker-0 to broker-2 (borker-3 on standby).
So e.g. partiton-0 was on broker-0 only, partition-1 on broker-1 only and partition-2 on broker-2 only. This resulted in the mentioned value of 3, which was really only 1 in my case.

tldr ;)
The approach in this PR is, to give an optimistic value for the replica count (max. value of nodes for a partition) and a pessimistic one for in-sync replicas (min. value of nodes found for a partition). So when the cluster is healthy both values should express the configured topic values, if one of the brokes is down on the other hand, in-sync replicas will adjust accordingly.

**Comment by tchiotludo** [comment]
That make sense !
Thanks for clarifying and for the fix 👍

**Comment by xpr3sso** [comment]
Glad to help.
Thanks for your great tool :)

**Comment by tchiotludo** [comment]
Thanks a lot 👍
